src/components/Popup/Popup.stories.tsx

```diff
@@ -103,9 +103,14 @@ Popups.args = {
       >
         <img
           src="https://static.vscdn.net/images/career_hub/upskilling/more_projects.svg"
-          style={{ height: 172, margin: 4, width: 'calc(100% - 8px)' }}
+          style={{
+            height: 172,
+            margin: 4,
+            width: 'calc(100% - 8px)',
+            zIndex: 0,
+          }}
         />
-        <div className="octuple">
+        <div className="octuple" style={{ zIndex: 1 }}>
```

Comment:
why was this necessary? possible code smell?

Reply:
Yes, need to investigate the `Stack` component behavior. May be isolated to Surface UI wrappers only.  Tracking with ENG-48127.